firestore-bigquery-export/functions/stress_test/count.js

```diff
@@ -0,0 +1,33 @@
+const admin = require("firebase-admin");
+
+// Initialize Firebase Admin with your credentials
+// Make sure you've already set up your Firebase Admin SDK
+admin.initializeApp({
+  projectId: "vertex-testing-1efc3",
```

Comment:
```suggestion
  projectId: "PROJECT_ID",
```

Reply:
oop, will tidy this up in a new commit later